app/(authenticated)/layout.tsx

```diff
@@ -31,8 +33,12 @@ export default async function Layout({
       >
         <NavItems />
       </div>
-
-      {children}
+      <LanguageProvider>
+        {children}
+        <div className="fixed top-4 right-4 z-50">
+          <LanguageSwitcher />
+        </div>
```

Comment:
Instead of fixed positioning can we make this a row at the top and justify-end?

Reply:
<img width="1374" alt="image" src="https://github.com/FlatFilers/products-show/assets/3760568/c42c537b-8444-490a-85dc-66c7e3d2ba4a">
like in hcm

Reply:
This seems harder than it should be with the way the layouts are. I'll keep working at it.

Reply:
mainly because of the different bg colors on home page vs the other pages.